docs/datasync/conflict-resolution-strategies.md

```diff
@@ -49,6 +49,11 @@ This strategy ensures that updates always resolve to whatever the client had sen
 This strategy ensures that in the event of a update conflict, the client's update will never overwrite any field that has changed since the client last fetched it. If the object has been deleted in the database, the client will be notified of it by way of a `ConflictError`. For delete conflicts, the client is informed of the conflict via a `ConflictError`.
 
 
+### ThrowOnConflict
+
+This strategy throws a `ConflictError` every time a conflict occurs, so that the client can be informed about the conflicts and can possibly resolve the conflicts.
```

Comment:
```suggestion
This strategy throws a `ConflictError` every time a version field from the client is different from than server-side version. The client can be informed about potential conflict and can resolve it if needed.
```

Reply:
This strategy currently only throws if there is conflict i.e. server and client change same fields, should I change it?

Reply:
Actually this is wrong. Our implementation will do merge.

Reply:
Yep. Still if you can rephrase it a little bit